beeline/propagation/test_w3c.py

```diff
@@ -3,53 +3,53 @@
 import beeline.propagation.honeycomb as hc
 import beeline.propagation.w3c as w3c
 
-traceparent_header = "00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-00"
-tracestate_header = "foo=bar,bar=baz"
+_TEST_TRACEPARENT_HEADER = "00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-00"
+_TEST_TRACESTATE_HEADER = "foo=bar,bar=baz"
 
-headers = {
-    "traceparent": traceparent_header,
-    "tracestate": tracestate_header
+_TEST_HEADERS = {
+    "traceparent": _TEST_TRACEPARENT_HEADER,
+    "tracestate": _TEST_TRACESTATE_HEADER
 }
-trace_id = "0af7651916cd43dd8448eb211c80319c"
-parent_id = "b7ad6b7169203331"
-trace_flags = "00"
-tracestate = "foo=bar,bar=baz"
+_TEST_TRACE_ID = "0af7651916cd43dd8448eb211c80319c"
+_TEST_PARENT_ID = "b7ad6b7169203331"
+_TEST_TRACE_FLAGS = "00"
+_TEST_TRACESTATE = "foo=bar,bar=baz"
 
 
 class TestW3CMarshalUnmarshal(unittest.TestCase):
     def test_roundtrip(self):
         '''Verify that we can successfully roundtrip (marshal and unmarshal)'''
-        trace_fields = {"traceflags": trace_flags,
-                        "tracestate": tracestate}
+        trace_fields = {"traceflags": _TEST_TRACE_FLAGS,
+                        "tracestate": _TEST_TRACESTATE}
 
-        pc = PropagationContext(trace_id, parent_id, trace_fields)
+        pc = PropagationContext(_TEST_TRACE_ID, _TEST_PARENT_ID, trace_fields)
 
-        traceparent_header = w3c.marshal_traceparent(pc)
-        self.assertEquals(traceparent_header, traceparent_header)
+        _TEST_TRACEPARENT_HEADER = w3c.marshal_traceparent(pc)
+        self.assertEquals(_TEST_TRACEPARENT_HEADER, _TEST_TRACEPARENT_HEADER)
 
-        tracestate_header = w3c.marshal_tracestate(pc)
-        self.assertEquals(tracestate_header, tracestate_header)
+        _TEST_TRACESTATE_HEADER = w3c.marshal_tracestate(pc)
+        self.assertEquals(_TEST_TRACESTATE_HEADER, _TEST_TRACESTATE_HEADER)
```

Comment:
I still think we need to assign a locally scoped variable from the `w3c.marshal_*` calls and compare that against the expected constant.

```
        marshalled_traceparent = w3c.marshal_traceparent(pc)
        self.assertEquals(_TEST_TRACEPARENT_HEADER, marshedled_traceparent)

        marshalled_tracestate = w3c.marshal_tracestate(pc)
        self.assertEquals(_TEST_TRACESTATE_HEADER, marshalled_tracestate)
```